### PR TITLE
added demo for list and type

### LINKED ISSUE #12: Add List Example to Doc
According to the doc, there is a list attribute that can be used to define the elements of a list, but there is no example that shows how to do this in the documentation.

### PR DESCRIPTION
**Pull request by e111077**
resolves #12

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -28,6 +28,14 @@
     The <code>value</code> stored at <code>key="info"</code> is <code><meta-test></meta-test></code>.
   </div>
 
+  <div class="vertical-section vertical-section-container centered">
```
Comment (rictic) [inline demo/index.html]: This is an existing issue, but here and in the above card, consider removing the `vertical-section-container` class.

As is,
the text is
very
vertical
and hard
to read.

If a bit poetic.

File: demo/index.html [hunk 2]
```diff
@@ -40,6 +48,22 @@
 
     });
 
+    Polymer({
+      is: 'type-one',
+
+      ready: function() {
+        this.textContent = new Polymer.IronMetaQuery({type: "type1"}).list;
```
Comment (rictic) [inline demo/index.html]: Consider calling `JSON.stringify` on the list to make it clearer to the reader that it's a list.

File: demo/index.html [hunk 3]
```diff
@@ -28,6 +28,14 @@
     The <code>value</code> stored at <code>key="info"</code> is <code><meta-test></meta-test></code>.
   </div>
 
+  <div class="vertical-section vertical-section-container centered">
+    <iron-meta type="type1" key="a" value="Polymer"></iron-meta>
```
Comment (rictic) [inline demo/index.html]: Consider adding a heading mentioning that you can also get list values.

**Comment by e111077** [comment]
Added headers and removed vertical text; here's how it should look:

screenshot from 2016-01-28 11 36 51

**Comment by rictic** [comment]
LGTM!

**Pull request merged**